web/views/api/campaign_view.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Jingle.API.CampaignView do
+  use Jingle.Web, :view
+
+  def render("index.json", %{conn: conn, campaigns: campaigns}) do
+    %{
+      count: length(campaigns),
+      total: length(campaigns),
+      _embedded: %{
```

Comment:
Someday we should abstract some of this HAL-generation logic, so we can do things like:

```elixir
resp(length(campaigns))
|> embed("prx:items", the_campaigns_json)
|> link("self", %{"http://what.ever"})
```

But it is not this day!

Reply:
👍  ⚔️